### PR TITLE
working language switcher, needs styling

### PR DESCRIPTION
**Pull request by petzlux**
@pgiraud can you work on the styling ?

### CODE REVIEW

**Comment by jaykayone** [comment]
I think the main question here is "how to integrate css from mockup templates into prod.

**Comment by pgiraud** [comment]
There's no real procedure on how to integrate css from design templates. The HTML/CSS from the design templates cannot be reused directly and have to be readapted to bootstrap/angular context.
The documentation on how to create a panel is actually the code I pushed in the project for the other tools. @petzlux already did the job.
I'll take a few minutes to help on the styling. I'll come back with a patch.

**Comment by pgiraud** [comment]
So, here's what I did.
I rebased the branch onto master.
Then I updated the html/less files so that it looks like in the design templates.
This is probably not perfect and we will eventually need to clean the less files a bit.
Can you please test it and review?

note: you'll probably need to reset your local branch:
```
git reset --hard origin/language_switcher
```

**Comment by petzlux** [comment]
Added autoclose of menu if language is switched , not sure if this Angular best practice, probably will get told of by @elemoine  :-)

**Comment by elemoine** [comment]
> Added autoclose of menu if language is switched , not sure if this Angular best practice, probably will get told of by @elemoine :-)

I'll have a look.

File: geoportailv3/templates/index.html [hunk 1]
```diff
@@ -26,10 +26,26 @@
           <a href class="navbar-brand" translate>logo</a>
         </div>
 
-        <ul class="nav navbar-nav pull-right">
+        <ul class="nav navbar-nav pull-right" ngeo-btn-group>
```
Comment (elemoine) [inline geoportailv3/templates/index.html]: You don't need `ngeo-btn-group` for now.

**Comment by elemoine** [comment]
> Added autoclose of menu if language is switched , not sure if this Angular best practice, probably will get told of by @elemoine :-)

It looks good to me.

**Pull request closed without merging**